tests/_runtime/test_virtual_file.py

```diff
@@ -185,19 +189,21 @@ def create_vfile(arg):
     # 0 references because HTML not bound to a variable
     # NB: this test may be flaky! refcount decremented when `__del__` is called
     # but we can't rely on when it will be called.
-    k.run([exec_req.get("gc.collect()")])
+    await k.run([exec_req.get("gc.collect()")])
     assert ctx.virtual_file_registry.refcount(vfile) == 0
 
     # this should dispose the old vfile (because its refcount is 0) and create
     # a new one
-    k.run([make_vfile])
+    await k.run([make_vfile])
     # the previous vfile should not be in the registry
     assert vfile not in ctx.virtual_file_registry.registry
     assert len(ctx.virtual_file_registry.registry) == 1
 
 
-def test_cached_vfile_disposal(k: Kernel, exec_req: ExecReqProvider) -> None:
-    k.run(
+async def test_cached_vfile_disposal(
```

Comment:
i thought we had to add `@pytest.mark.asyncio` to all the tests. was there a config change that allowed us to remove this? should we remove other instances of `@pytest.mark.asyncio`?

Reply:
There was a config change that enables us to omit this. It's probably fine to keep the existing marks.

https://pytest-asyncio.readthedocs.io/en/latest/concepts.html#auto-mode

Configured in pyproject.toml